lib/crystalball/logger.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'logger'
+
+module Crystalball
+  class << self
```

Comment:
Make it a separate module and extend `Crystalball` module with it

Reply:
Would you care to clarify the point? It would have the same outcome with different syntax, and would not be extended elsewhere.